wfsim/core.py

```diff
@@ -373,7 +373,7 @@ def photon_channels(positions, n_photons, config, s1_pattern_map):
         return _photon_channels
 
     @staticmethod
-    def photon_timings(t, n_photons, recoil_type, config, phase):
+    def photon_timings(t, n_photons, recoil_type, config, phase, channels=None,positions=None,resource=None):
```

Comment:
(Small comment) Since we're here, would be nice to extend a bit on the args description in the docstring below, plus of course add the new args you coded.

Reply:
Done